semgrep/semgrep/rule.py

```diff
@@ -267,7 +268,27 @@ def mode(self) -> str:
         return self._mode
 
     @property
-    def languages(self) -> List[Language]:
+    def sarif_severity(self) -> str:
+        """
+        SARIF v2.1.0-compliant severity string.
+
+        See https://github.com/oasis-tcs/sarif-spec/blob/a6473580/Schemata/sarif-schema-2.1.0.json#L1566
+        """
+        mapping = {"INFO": "note", "ERROR": "error", "WARNING": "warning"}
+        return mapping[self.severity]
+
+    @property
+    def sarif_tags(self) -> Iterator[str]:
+        """
+        Tags to display on SARIF-compliant UIs, such as GitHub security scans.
+        """
+        if "cwe" in self.metadata:
+            yield self.metadata["cwe"]
+        if "owasp" in self.metadata:
+            yield f"OWASP-{self.metadata['owasp']}"
```

Comment:
I don't know what this section is---it was added when I merged develop

Reply:
This is from a bad merge, can you remove it?